server/app/views/components/AccordionFactory.java

```diff
@@ -0,0 +1,56 @@
+package views.components;
+
+import static j2html.TagCreator.button;
+import static j2html.TagCreator.div;
+
+import j2html.tags.ContainerTag;
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import views.style.ReferenceClasses;
+
+/**
+ * Utility class for building an accordion. This is a modified version of the USWDS accordion
+ * component here: https://designsystem.digital.gov/components/accordion/.
+ */
+public final class AccordionFactory {
```

Comment:
suggestion (non-blocking): I have a slight preference for editing the existing class rather than creating a new one since there's some overlap, but I'm curious to know your thinking.

Reply:
Essentially, I did edit the existing class, but, because I changed the name of the class (see my reasoning above), the filename had to be changed and version control sees this as a completely new file.